### ISSUE TITLE
Trouble dealing with multivalued attributes

### ISSUE THREAD

**Issue by thijskh** [OpenConext/OpenConext-profile#46]
Profile has a problem dealing with multivalued attributes. Logging into profile with a common set of attributes therefore fails with:
```
request.CRITICAL: Uncaught PHP Exception UnexpectedValueException: "AttributeDefinition "eduPersonEntitlement" has a single-value multiplicity, got "2" values" at /opt/openconext/OpenConext-profile-0.2.0/vendor/surfnet/stepup-saml-bundle/src/SAML2/Attribute/Attribute.php line 44
```

Most likely, the best way to fix this, is to upgrade to stepup-saml-bundle >= 2 which removed the concept of attribute multiplicity. Profile seems to work with this, except for attributes like this:

```
            <saml:Attribute Name="urn:mace:dir:attribute-def:eduPersonTargetedID"
                            NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:uri"
                            >
                <saml:AttributeValue>
                    <saml:NameID Format="urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified">urn:collab:person:surfnet.nl:kinkhorst</saml:NameID>
                </saml:AttributeValue>
            </saml:Attribute>
```
which result in doubly nested arrays which the twig templates cannot deal with currently.

**Comment by arothuis** [comment]
It does look like an error in how multiplicity is dealt with and `simple-saml-bundle >= 2` indeed fixes this.
The template could then be rendered by doing a nested for-loop.

However, I see an empty `DOMNodeList` as a value for the `eduPersonTargetedID` in my reproduction of the issue and it will therefore show up blank when the template-solution would be implemented.

I find this strange for two reasons:

1. A `DOMNodeList` is IMHO not really a useful representation of an attribute. Other attributes are scalar/strings, while a `DOMNodeList` is specific for how XML parsing is handled internally.
2. The `DOMNodeList` should contain a NameID in my reproduction of the issue, because I see a NameID in the SAML request.

At first sight, this is due to how `simple-saml-bundle` deals with this kind of nesting. An adequate solution would require some more research.

**Comment by thijskh** [comment]
Thanks. For profile, I would be fine with just not displaying these kinds of attributes at all (for now), as they are generated by OpenConext. If we upgrade the lib and skip any attribute that is not a simple type, we have a useful and functioning profile.

The issue in `stepup-saml-bundle` is indeed something to discuss there, perhaps turn that into an Issue?

**Comment by arothuis** [comment]
This is resolved in #49
